templates/pulpcore-docker-apache.conf.erb

```diff
@@ -1,6 +1,10 @@
 <Location /pulpcore_registry/v2/>
-  SSLRequire %{SSL_CLIENT_S_DN_CN} eq "admin" 
+  RequestHeader set REMOTE_USER ""
+  RequestHeader set REMOTE_USER "%{SSL_CLIENT_S_DN_CN}s" env=SSL_CLIENT_S_DN
 </Location>
 
-ProxyPass /pulpcore_registry/v2/ http://<%= scope['pulpcore::content_host'] %>:<%= scope['pulpcore::content_port']%>/v2/
-ProxyPassReverse /pulpcore_registry/v2/ http://<%= scope['pulpcore::content_host'] %>:<%= scope['pulpcore::content_port'] %>/v2/
+ProxyPass /pulpcore_registry/v2/ http://<%= scope['pulpcore::api_host'] %>:<%= scope['pulpcore::api_port']%>/v2/
+ProxyPassReverse /pulpcore_registry/v2/ http://<%= scope['pulpcore::api_host'] %>:<%= scope['pulpcore::api_port'] %>/v2/
+
+ProxyPass /pulp/container/ http://<%= scope['pulpcore::content_host'] %>:<%= scope['pulpcore::content_port']%>/pulp/container/
```

Comment:
How about this location? Should this also be receiving REMOTE_USER?

Reply:
nope, it doesn't.  all the auth to that endpoint is handled by the generated tokens passed via the redirect from the /v2/ endpoint in the api app

Reply:
actually, it looks like a crazy request like this:

 curl https://`hostname`/pulp/container/../api/v3/status/
works, but i can't get something like this:

curl "https://`hostname`/pulp/container/../api/v3/tasks/" -H 'REMOTE_USER: admin'

to work.  it still complains about auth.     It seems like apache is doing the right thing here and recognizing that '../api/v3/tasks' actually matches the '/pulp/api/v3/' Location.  If i comment out the "RequestHeader set REMOTE_USER " line from the /pulp/api/v3/ location, it then works. 

So yes, this is good as is

Reply:
Ok. In https://github.com/theforeman/puppet-pulpcore/pull/105 I'll probably revisit this and see how it turns out.